MultithreadedProgram/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace MultithreadedProgram
+{
+    class Constants
+    {
+        public const string ThreadNumberAsker = "Please enter the number of threads";
+        public static int LenghtOfConsoleColorEnum = Enum.GetNames(typeof(ConsoleColor)).Length;
```

Comment:
The class is named as Constants, so the variables in it should be constant not static.

Reply:
Rename this ColorEnumLength